### ISSUE TITLE
Time to design a logo for khmer?

### ISSUE THREAD

**Issue by qingpeng** [ged-lab/khmer#982]
or wait until we adopt the new name? :)

**Comment by mr-c** [comment]
Yes, we need a logo for oxli! This will fulfill item 2 on https://github.com/ged-lab/khmer/issues/291#issue-27482199
